swift/README.md

```diff
@@ -0,0 +1,3 @@
+# Workflow
```

Comment:
Are you planning to add release instructions? We've got them for [kotlin](https://github.com/square/workflow/blob/master/kotlin/RELEASING.md), and they're linked to from the root readme. If you add them, just link from there as well.

Reply:
Not yet, but we should add them as we figure it out.